webui/react/package.json

```diff
@@ -96,7 +96,6 @@
     "@types/node": "^18.11.9",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.8",
-    "@types/react-router-dom": "^5.3.3",
```

Comment:
q: is it ok to uninstall this? 
Does `react-router-dom` v6 supports type natively?

Reply:
Yes -- as of v6, react-router and react-router-dom are written in typescript and distribute their own types. You can check locally in your node_modules.